definitions/ext-rfscanner/golden_metric.yml

```diff
@@ -0,0 +1,39 @@
+# Determine metrics for VALET
+
+
+latency:
+  title: Latency  (ms)
+  displayAsValue: true
+
+
+  query:
+    select: average(totalTime) AS 'Latency'
```

Comment:
nit: you can remove the `AS ...` we will use the key of the metric (`latency` here). Same for the others